File: benchmarks/loop_sum.c
#include <stdio.h>
int main(int argc, char **argv) {
    (void)argv;
    volatile long limit = 10000000L + (argc - 1);
    long s = 0;
    for (long i = 1; i <= limit; ++i) s += i;
    printf("sum=%ld\n", s);
    return 0;
}

File: benchmarks/loop_sum.rs
fn main() {
    let argc = std::env::args().count() as i64;
    let limit = std::hint::black_box(10_000_000_i64 + (argc - 1));
    let mut s: i64 = 0;
    let mut i: i64 = 1;
    while i <= limit { s += i; i += 1; }
    println!("sum={}", s);
}
